compiler/src/dotty/tools/dotc/printing/PlainPrinter.scala

```diff
@@ -208,7 +208,10 @@ class PlainPrinter(_ctx: Context) extends Printer {
         else {
           val constr = ctx.typerState.constraint
           val bounds =
-            if (constr.contains(tp)) constr.fullBounds(tp.origin)(ctx.addMode(Mode.Printing))
+            if (constr.contains(tp)) {
+              val ctx0 = ctx.addMode(Mode.Printing)
```

Comment:
Why not leave it inline? `ctx.addMode(Mode.Printing).typeComparer.fullBounds(tp.origin)`